### PR TITLE
Revert particle momentum z component

### PR DESCRIPTION
**Pull request by miweberSMI**

### CODE REVIEW

File: EVGEN/AliGenReadersEMD.h [hunk 1]
```diff
@@ -33,13 +33,15 @@ class AliGenReadersEMD : public AliGenReader
     void TrackAll() {fPcToTrack = kAll;}
     void SetStartEvent(Int_t nev) {fStartEvent = nev;}
     void SetNtupleName(TString s) {fNtupleName=s;}
+    void SetInvertPz() {fInvertPz=kTRUE;}
```
Comment (qgp) [inline EVGEN/AliGenReadersEMD.h]:
```suggestion
    void SetInvertPz(Bool_t invert = kTrue) {fInvertPz=invert;}
    Bool_t GetInvertPz() const { return fInvertPz; }
```

Comment (miweberSMI) [reply EVGEN/AliGenReadersEMD.h]: Hi @qgp,

thanks for this suggestion. Changed accordingly.

Cheers,
Michael

**Comment by qgp** [comment]
+1

**Comment by miweberSMI** [comment]
Hi @qgp, 

could you trigger once again?

Thanks and cheers,
Michael

**Comment by qgp** [comment]
+1

**Pull request merged**